docs/features/backup_and_restore.md

```diff
@@ -3,12 +3,190 @@ id: backup_and_restore
+
+* If the Postgres database cluster exists/is still online, take it
+  offline/remove access.
+* If needed, recover the DynamoDB tables as noted in the
+  [DyanmoDb](./backup_and_restore#dynamodb) section of this document.
+* Redeploy a new database cluster from your backup, matching as closely as possible to the DynamoDB
+  restore time.   See [AWS's
+  PIT recovery
+  instructions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PIT.html)
+  and [DB Snapshot recovery
+  instructions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RestoreFromSnapshot.html),
+  or the examples below for more information.
+* Configure your Cumulus deployment to utilize the new database cluster and re-deploy
+* Run DynamoDB/RDS reconciliation tools and resolve any discrepancies.
+
+##### cumulus-rds-tf examples
+
+The following sections provide a walk through of a few recovery scenarios for the provided `cumulus-rds-tf`
+serverless module.
+
+***Point In Time Recovery***
+
+If you need recovery that exceeds the 1-day granularity of AWS's snapshots, you
+either must create and manually manage snapshots, or use Point In Time
+Recovery(PITR) if you still have the original cluster available.
+
+Unfortunately as terraform does not yet support RDS PITR (see:
+[github terraform-provider issue #5286](https://github.com/terraform-providers/terraform-provider-aws/issues/5286)),
+this requires a manual procedure.
+
+If you are using the `cumulus-rds-tf` module to deploy an RDS Aurora Serverless
+Postgres instance, the following procedure can be used to successfully spin up a duplicate
+cluster from backup in recovery scenarios:
+
+1. Halt all ingest and remove access to the database to prevent Core processes from
+   writing to the old cluster.
+2. Using the AWS CLI (see [AWS PITR
+   documentation](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_PIT.html)
+   for console instructions), run the following command, making *certain* to use
+   the same subnet groups and vpc-security-group IDs from your Core deployment:
+
+```bash
+aws rds restore-db-cluster-to-point-in-time --source-db-cluster-identifier "<cluster_needing_restoration>" --restore-to-time "<time>" --vpc-security-group-ids "<security group 1>" "<security group 2>" --copy-tags-to-snapshot --db-cluster-identifier "<new cluster identifier>" --db-subnet-group-name "<db-subnet-group>"
+```
+
+You can get the configuration vales from the [RDS
+console](https://console.aws.amazon.com/rds/) *or* by running the following
+command and parsing the outputs:
+
+```bash
+aws rds describe-db-clusters
+```
+
+* cluster_needing_restoration -- the name of the database cluster you're
+  restoring *from* (`DBClusterIdentifier` from the AWS RDS CLI output)
+* time - The time in UTC format (e.g. 2015-03-07T23:45:00Z)
+* security_group_n - the security group IDs from your original deployment
```

Comment:
These fields have dashes, underscores, and spaces. Is that just how it is?

Reply:
No, it's just sloppy.  I'll sort it to use underscores.

Reply:
f4d093f05